.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Thank you for sending in this pull request. Please make sure you take a look at the [contributing file](CONTRIBUTING.md). Here's a few things for you to consider out on this pull request:
```

Comment:
"Here's a few things for you to consider in this pull request:" ?

Reply:
Oops, thanks.